frontend/EmpUI/src/services/leave-service.ts

```
@@ -28,6 +28,13 @@ export const leaveService = baseService.injectEndpoints({
       },
       invalidatesTags: ["leaves", "leave"],
     }),
+    empLeave: builder.query<User, string>({
+      query: (empId) => ({
+        url:  API_END_POINTS.leave + `?EmployeeId=${empId}`,
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)